cleanrl/tqc_continuous_action.py

```diff
@@ -0,0 +1,363 @@
+        help="if toggled, cuda will be enabled by default")
+    parser.add_argument("--track", type=lambda x: bool(strtobool(x)), default=False, nargs="?", const=True,
+        help="if toggled, this experiment will be tracked with Weights and Biases")
+    parser.add_argument("--wandb-project-name", type=str, default="cleanRL",
+        help="the wandb's project name")
+    parser.add_argument("--wandb-entity", type=str, default=None,
+        help="the entity (team) of wandb's project")
+    parser.add_argument("--capture-video",
+        type=lambda x: bool(strtobool(x)), default=False, nargs="?", const=True,
+        help="weather to capture videos of the agent performances (check out `videos` folder)")
+
+    # Algorithm specific arguments
+    parser.add_argument("--env-id", type=str, default="HopperBulletEnv-v0",
+        help="the id of the environment")
+    parser.add_argument("--total-timesteps", type=int, default=5000000,
+        help="total timesteps of the experiments")
+    parser.add_argument("--buffer-size", type=int, default=int(1e6),
+        help="the replay memory buffer size")
+    parser.add_argument("--gamma", type=float, default=0.99,
+        help="the discount factor gamma")
+    parser.add_argument("--tau", type=float, default=0.005,
+        help="target smoothing coefficient (default: 0.005)")
+    parser.add_argument("--batch-size", type=int, default=256,
+        help="the batch size of sample from the reply memory")
+    parser.add_argument("--handle-timeout-termination",
+        type=lambda x: bool(strtobool(x)), default=False,
+        help="treat TimeLimit.truncated == True as done == False")
+    parser.add_argument("--n-quantiles", type=int, default=25,
+        help="the number of quantiles used for each Q Network")
+    parser.add_argument("--n-critics", type=int, default=5,
+        help="the number of Q Networks")
+    parser.add_argument("--use-paper-n-quantiles-to-drop",
+        type=lambda x: bool(strtobool(x)),
+        default=True,
+        help="number of quantiles to drop")
+    parser.add_argument("--n-quantiles-to-drop", type=int, default=2,
+        help="number of quantiles to drop")
+    parser.add_argument("--learning-starts", type=int, default=5e3,
+        help="timestep to start learning")
+    parser.add_argument("--actor_adam_lr", type=float, default=3e-4,
+        help="the learning rate of the policy network optimizer")
+    parser.add_argument("--critic_adam_lr", type=float, default=3e-4,
+        help="the learning rate of the Q network network optimizer")
+    parser.add_argument("--alpha_adam_lr", type=float, default=3e-4,
+        help="the learning rate to tune target entropy")
+    args = parser.parse_args()
+    # fmt: on
+    return args
+
+
+def layer_init(layer, std=np.sqrt(2), bias_const=0.0):
+    torch.nn.init.orthogonal_(layer.weight, std)
+    torch.nn.init.constant_(layer.bias, bias_const)
+    return layer
```

Comment:
Does the original paper use PPO-style initialization? If the paper doesn't use it, we should not use it by default.  The implementations at CleanRL should try to match the original implementations whenever possible.

Reply:
Just checked the author's implementation. The paper does not use any initialization.